apps/election-manager/src/components/ImportCVRFilesModal.tsx

```diff
@@ -135,8 +165,56 @@ const ImportCVRFilesModal: React.FC<Props> = ({ isOpen, onClose }) => {
     }
   }, [usbDriveStatus])
 
+  if (currentState === ModalState.ERROR) {
+    return (
+      <Modal
+        isOpen={isOpen}
+        content={
+          <Prose>
+            <h1>Error</h1>
+            <p>
+              There was an error reading the content of the file{' '}
+              <strong>{castVoteRecordFiles?.lastError?.filename}</strong>:{' '}
+              {castVoteRecordFiles?.lastError?.message}. Please ensure this file
+              only contains valid CVR data for this election.
+            </p>
+          </Prose>
+        }
+        onOverlayClick={onClose}
+        actions={
+          <React.Fragment>
+            <LinkButton onPress={onClose}>Close</LinkButton>
+          </React.Fragment>
+        }
+      />
+    )
+  }
+
+  if (currentState === ModalState.DUPLICATE) {
+    return (
+      <Modal
+        isOpen={isOpen}
+        content={
+          <Prose>
+            <h1>Duplicate File</h1>
+            <p>
+              The selected file was ignored as a duplicate of a previously
+              imported file.
```

Comment:
This works, but I worry that this flow may cause some confusion. 

Questions a user may have:
- Wait, what file did I select and which file was it a duplicate of?
- Why didn't the previous modal inform me the file was a duplicate?
- Why is the only option to close this modal? Why can't I go back to the previous modal?

It's a bit more work to make the flow more delightful, but we should iterate towards making the flow delightful and informative. As it is right now, this flow just kinda feels like "You did something wrong. Click here to acknowledge." 😛

Reply:
Yeah fair, will keep thinking about how to make this better.